examples/text/plot_hashing_vs_dict_vectorizer.py

```diff
@@ -66,8 +65,7 @@ def token_freqs(doc):
 
 print(__doc__)
 print("Usage: %s [n_features_for_hashing]" % sys.argv[0])
-print("    The default number of features is 2**18.")
-print()
+print("    The default number of features is 2**18.\n")
```

Comment:
what happen if:

```suggestion
print("\tThe default number of features is 2**18.\n")
```

Reply:
Actually i was more on focussing on removing the import statements .
Thanks !

Reply:
yes sure! Just are suggestions :-)